docs/source/notebooks/regridding.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p><code>wfs_url = 'http://pavics.ouranos.ca/geoserver/wfs</code>  keep  the hostname <code>boreas.ouranos.ca</code>  to avoid it being replaced.  We are not able to provision a working geoserver yet, see https://github.com/Ouranosinc/pavics-sdi/issues/183</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/pavics-sdi/pull/216/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>